doc/rtd/index.rst

```diff
@@ -15,10 +15,11 @@ largest scale on DOE HPC systems.
     Storm Tracks Generated by TECA
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 2
```

Comment:
Unless you have a compelling reason for the change I'd prefer to leave at 3 for now.